README.md

```diff
@@ -26,10 +26,11 @@ $ npm install signalfx
 
 ### Supported Node.js versions
 
-| Version | Node.js       |
-| ------- | ------------- |
-| `7.4.x` | `>=8.0.0 <18` |
-| `7.3.1` | `>=8.0.0 <11` |
+| Version | Node.js        |
+| ------- | -------------- |
+| `8.x.x` | `>=11.0.0 <18` |
```

Comment:
The comment in `CHANGELOG` states Node <12.10 was dropped, should this be `>=12.10.0 <18` here?